frontend/docs/docs/user-guide/presentation-sharing.md

```diff
@@ -0,0 +1,24 @@
+# Collection Presentation & Sharing
+
+## Sharing Collections
+
+Collections are private by default, but can be made sharable either with an unlisted link, or displayed in your org's public gallery for everyone to discover. Sharing settings can be found within the **Sharing** tab of the Collection Settings dialog. Public collections will not be visible unless your org has enabled its [public collections gallery](org-settings.md#public-collections-gallery).
+
+After a collection has been made public the collection can also be embedded into other websites using the provided embed code. Un-sharing the collection will break any previously shared links and remove it from your gallery.
+
+For further resources on embedding archived web content into your own website, see the [ReplayWeb.page docs page on embedding](https://replayweb.page/docs/embedding).
+
+## Initial View
+
+Set the Initial View to control what visitors see first when viewing a collection. The initial view can either be a list of pages (ReplayWeb.page's default), or a single page from your collection such as a crawl start URL or index page.
+
+## Thumbnails
+
+The collection thumbnail image will be displayed as a card within your org's public gallery. This thumbnail can be edited from the **Presentation** tab of the Collection Settings dialog. Thumbnails can either be a screenshot of a page from the collection, or a placeholder provided by Webrecorder.
+
+??? question "Why don't the pages in my collection have thumbnails?"
+    If you have uploaded archived items captured with tools other than Browsertrix, some of the pages in your collection may not have screenshots available to use as thumbnails. To fix this, either re-crawl a page you wish to use as a thumbnail with Browsertrix and add it to the collection, or choose a placeholder thumbnail.
```

Comment:
Choosing a placeholder thumbnail doesn't fix the issue posed by the question "why don't the pages in my collection have thumbnails".
```suggestion
    If you have uploaded archived items captured with tools other than Browsertrix, some of the pages in your collection may not have screenshots available to use as thumbnails. To fix this, re-crawl the page you wish to use as a thumbnail with Browsertrix and add it to the collection.
```

Alternatively, maybe we don't show this question until we support thumbnail uploads?

Reply:
Objectively, sure it is sort of an aside, though I do see some value in offering all the options available (which is why I included that bit).  I'll merge it in if you feel strongly about it!

Uploading a thumbnail is probably a better eventual solution to offer?  I would like to update this if / when that gets added!